core/src/main/java/io/ran/AutoWrapper.java

```diff
@@ -0,0 +1,45 @@
+package io.ran;
+
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.util.CheckClassAdapter;
+
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AutoWrapper {
+	private static Map<Class, Class> wrapped = new HashMap<>();
```

Comment:
```suggestion
	private static Map<Class<?>, Class<?>> wrapped = new HashMap<>();
```

Reply:
I'm pretty sure this would cause type problems